terraform/lambda.tf

```diff
@@ -47,6 +47,7 @@ resource "aws_lambda_function" "integration_service_lambda" {
     variables = {
       ENV = var.environment_name
       PENNSIEVE_DOMAIN = data.terraform_remote_state.account.outputs.domain_name,
+      RDS_PROXY_ENDPOINT = data.terraform_remote_state.pennsieve_postgres.outputs.rds_proxy_endpoint
```

Comment:
@jwagenaar - had not merged this as yet, so will add this missing variable as part of this.